cmd/eksctl/main.go

```diff
@@ -90,13 +91,18 @@ func main() {
 	loggerLevel := rootCmd.PersistentFlags().IntP("verbose", "v", 3, "set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging")
 	colorValue := rootCmd.PersistentFlags().StringP("color", "C", "true", "toggle colorized logs (valid options: true, false, fabulous)")
 
+	logBuffer := new(bytes.Buffer)
+
 	cobra.OnInitialize(func() {
-		initLogger(*loggerLevel, *colorValue)
+		initLogger(*loggerLevel, *colorValue, logBuffer)
 	})
 
 	rootCmd.SetUsageFunc(flagGrouping.Usage)
 
 	if err := rootCmd.Execute(); err != nil {
+		if dumpErr := dumpLogsToDisk(logBuffer, err.Error()); dumpErr != nil {
```

Comment:
I'm wondering if we should put this in SIGTERM with a signal catcher so it dumps the logs in case eksctl was terminated. Not necessary in this PR, I'm just musing about it. :)